app/validation/validators.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime
+
+from app.validation.error_messages import error_messages
+
+from wtforms import validators
+
+
+class IntegerCheck(object):
```

Comment:
Thought about this some more, can we not derive from the WTForms validator and just override/catch/re-throw the exception with our message to re-use as much of the existing validator as possible?